tests/README.md

```diff
@@ -5,6 +5,7 @@ Before running the tests, start an Etherpad instance on your machine.
 ## Frontend
 
 To run the frontend tests, point your browser to `<yourdomainhere>/tests/frontend`
+Also for nested pad tests, point your browser to `<yourdomainhere>/tests/nested-pad/frontend`
```

Comment:
this wont work.  The automated test runner wont know to run these tests.  The tests need to run "as normal" part of our test runner.

Reply:
Removed it and merged with the original test